CHANGELOG.md

```diff
@@ -4,11 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [1.14.0] - 2021-08-12
+### Added
+- Introducing `fx.WithLogger` option that accepts a constructor that returns an 
+  `fxevent.Logger`. This option will override internal logger that will be used
+  to log internal events.
+- Exposing `fxevent` logging interface to implement if using `WithLogger` option
+  when supplying custom logger.
+
```

Comment:
We also added a report for hooks runtimes' as part of the error messages displayed on hook timeouts in  https://github.com/uber-go/fx/pull/746.